features/signal/child_workflow/feature.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+from temporalio import workflow
+from temporalio.client import WorkflowHandle
+
+from harness.python.feature import Runner, register_feature
+
+CHILD_WORKFLOW_INPUT = "child-input"
+CHILD_WORKFLOW_INITIAL_STATE = "1"
+SIGNAL_ARG = "2"
+
+
+@workflow.defn
+class Workflow:
+    @workflow.run
+    async def run(self) -> str:
+        child_wf = await workflow.start_child_workflow(
+            ChildWorkflow.run, CHILD_WORKFLOW_INPUT
+        )
+        await child_wf.signal("my_signal", SIGNAL_ARG)
+        await child_wf
+        return child_wf.result()
+
+
+@workflow.defn
+class ChildWorkflow:
+    def __init__(self) -> None:
+        self._state = CHILD_WORKFLOW_INITIAL_STATE
```

Comment:
```suggestion
        self._state: Optional[str] = None
```

Instead of constants, but this is pedantic, don't have to change

Reply:
Thanks, agree that's a cleaner form for the initial state. Taken.

Reply:
May want to apply to other features in this PR